src/PowerShellEditorServices/Services/DebugAdapter/Handlers/ConfigurationDoneHandler.cs

```diff
@@ -105,9 +105,13 @@ public Task<ConfigurationDoneResponse> Handle(ConfigurationDoneArguments request
         private async Task LaunchScriptAsync(string scriptToLaunch)
         {
             PSCommand command;
-            // Script could an actual script, or a URI to a script file (or untitled document).
-            if (!System.Uri.IsWellFormedUriString(scriptToLaunch, System.UriKind.RelativeOrAbsolute)
-                || ScriptFile.IsUntitledPath(scriptToLaunch))
+            if (System.IO.File.Exists(scriptToLaunch))
```

Comment:
I think this is going to break for remote files (e.g. you do an interactive `Enter-PSSession` and then `psedit ./something.ps1`) but that's also probably super broken atm anyway. Something we should revisit in the future, but definitely not a blocker to getting this fixed